homeassistant/components/landisgyr_heat_meter/manifest.json

```diff
@@ -7,7 +7,6 @@
   "ssdp": [],
   "zeroconf": [],
   "homekit": {},
-  "dependencies": [],
```

Comment:
Since this necessary for update to `prettier` but not to `python-typing-update` I suggest that you move `python-typing-update` to a separate PR.

Reply:
Um, this doesn't touch `prettier`..?

Reply:
Sorry - I guess it's `pre-commit-hooks` that checks the json duplicates.